SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2546,18 +2547,31 @@ void USpatialNetDriver::TryFinishStartup()
 {
 	if (IsServer())
 	{
+		// Limit Log frequency.
+		bool bShouldLogStartup = [this]() {
+			const uint64 WatchdogTimer = 5.0 / FPlatformTime::GetSecondsPerCycle64();
```

Comment:
I think we should make this an ini file configurable constant rather than a magic number in code

Reply:
Agreed, if we can somehow make this approach viable.